.saucetpl/.sauce/config.yml

```diff
@@ -1,18 +1,24 @@
 apiVersion: v1alpha
-metadata:
-  name: Feature XYZ
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: Release $CI_COMMIT_SHORT_SHA
-files:
-  - tests/example.test.js
-suites:
-  - name: "saucy test"
-    match: ".*.(spec|test).js$"
-image:
-  base: saucelabs/stt-playwright-jest-node
-  version: ##VERSION##
+kind: playwright
 sauce:
   region: us-west-1
+  concurrency: 1
+  metadata:
+    name: Testing Playwright Support
+    tags:
+      - e2e
+    build: "$BUILD_ID"
+playwright:
+  version: ##VERSION##
```

Comment:
Has the pipeline already been updated like cypress, so that it sets the numeric version here instead of `v.1.2.3`?

Reply:
Now it has https://github.com/saucelabs/sauce-playwright-runner/pull/58/commits/ace6791c49933463806c4c8dec23de68887c86ef
(Thanks for reminding).